controllers/masteruserrecord/sync_test.go

```diff
@@ -197,6 +196,7 @@ func TestSyncMurStatusWithUserAccountStatusWhenUpdated(t *testing.T) {
 	userSignup.Status = toolchainv1alpha1.UserSignupStatus{
 		CompliantUsername: "john",
 	}
+	userTier := commontier.NewUserTier(commontier.WithName("deactivate30"))
 	mur := murtest.NewMasterUserRecord(t, "john",
```

Comment:
Well... by default the user uses `deactivate30` tier. And in the context of this test the actual number of days in the tier is not that important. If we want to use another tier here (and not the default one) then we would need to make more changes in the test to use a specific tier instead of the default one.
Anyway, I added a comment https://github.com/codeready-toolchain/host-operator/pull/1077/commits/1d00d5ec4bd8583ff568a902b99d65341e1a5de2 to make it less confusing.

Reply:
Let me know if you still think we need to change it.

Reply:
I'm a bit confused why would you need more changes? If commit my two proposals then it will work (see that I added also `murtest.TierName("nodeactivation")` to set the correct tier. 
The reason why I proposed the change was that I got confused by:
* using `deactivate30` tier
* but expecting the context to contain the unlimited value
* and for the same setup expecting the error to contain `deactivate30` in the following test (which didn't make much sense after reading the previous test with the "unlimited" context)

it took me a while to go over it and understand why it is so.
As I mentioned above, the two changes that I proposed are enough to make it work and have the code easily understandable without any need of additional comments. 
That being said, the comment is fine, but my question is: How is it better compared to naming the tier correctly?

Reply:
> I'm a bit confused why would you need more changes?

I missed `murtest.TierName("nodeactivation")` in the suggested change. I didn't want to test the suggestions by simply committing the suggestions. So I modified the code locally only but missed that change.

Sorry for rushing it. Here is the PR to address it: https://github.com/codeready-toolchain/host-operator/pull/107